dali/test/python/test_external_source_impl.py

```diff
@@ -90,8 +95,9 @@ def make_torch_tensor(*args, **kwargs):
 
     def torch_cast(x, dtype):
         return x.type(dtype)
+
     cast_to = torch_cast
-    random_array = lambda shape: make_torch_tensor(np.random.ranf(shape))
+    random_array = lambda shape: make_torch_tensor(np.random.ranf(shape))  # noqa: E731
```

Comment:
```suggestion
    def random_array(shape):
        return make_torch_tensor(np.random.ranf(shape))
```
....if it will know what `make_torch_tensor` is, of course.